app/scripts/modules/amazon/loadBalancer/details/loadBalancerDetail.controller.js

```diff
@@ -25,24 +25,18 @@ module.exports = angular.module('spinnaker.loadBalancer.aws.details.controller',
     $scope.InsightFilterStateModel = InsightFilterStateModel;
 
     function extractLoadBalancer() {
-      if (!loadBalancer.vpcId) {
-        loadBalancer.vpcId = null;
-      }
-      $scope.loadBalancer = app.loadBalancers.data.filter(function (test) {
-        var testVpc = test.vpcId || null;
-        return test.name === loadBalancer.name && test.region === loadBalancer.region && test.account === loadBalancer.accountId && testVpc === loadBalancer.vpcId;
-      })[0];
+      let [appLoadBalancer] = app.loadBalancers.data.filter(function (test) {
```

Comment:
Yeah, exactly - it's using ES6 array destructuring. Since `filter` returns an array, this sets `appLoadBalancer` to the first item in the array (or undefined if the filter returns an empty collection).